aikau/src/main/resources/alfresco/layout/HorizontalWidgets.js

```diff
@@ -157,8 +171,17 @@ define(["alfresco/core/ProcessWidgets",
          // Split the full width between all widgets... 
          // We should update this to allow for specific widget width requests...
          this.visibilityRuleTopics = this.getVisibilityRuleTopics(this.widgets);
-         this.doWidthProcessing(this.widgets, true);
-         this.inherited(arguments);
+
+         // NOTE: Here we're very deliberately using JQuery to get the available width, this
+         //       is to address a very specific issue with Firefox failing when using iframes
+         //       and the dojo/dom-geometry code (in particular the getComputedStyle) - see AKU-692
+         var overallwidth = $(this.domNode).width();
```

Comment:
No, of course not ... would be good to try and use native over library though, if we can, for obvious reasons.